README.md

```diff
@@ -1,20 +1,171 @@
 ## Overview
 
 ### What is SBOM quality score
+A quality SBOM is one that is accurate, complete, and up-to-date. It should accurately reflect the components and dependencies used in the software application, including their version numbers and optionally any known vulnerabilities. In addition, it should be easily accessible and understandable by stakeholders, such as developers, security teams, and compliance officers.
 
+[Interlyk.io](mailto:hello@interlynk.io) has developed sbomqs to simplify the evaluation of SBOM quality for both producers and consumers. A higher score indicates greater usability of the SBOM contents.
 
+### SBOM Support
+We support SPDX and CycloneDX sbom formats, in various file formats.
 
-### Installation
+### Installation 
+Use the steps below to try out the tool. You will need
+golang version 1.19
 
+1. Clone the repo ```git clone git@github.com:interlynk-io/sbomqs.git```
+2. `cd` into `sbomqs` folder 
+3. make
+4. To test if the build was successful run the following command ```./build/sbomqs version```
 
-### Examples
-
+### Getting access to SBOMS
+For new users, we have listed a few places, where you could find sboms to play around with.
+- This repo has a [samples](https://github.com/interlynk-io/sbomqs/tree/main/samples) directory
+- If you use docker images and would like to generate an SBOM, follow the steps [here](https://docs.docker.com/engine/sbom/)
+- [Syft](https://github.com/anchore/syft) / [Trivy](https://github.com/aquasecurity/trivy) are open source tools which can be used to generate SBOM from containers as well as repos. 
+- Chainguard has created a repo called [bom-shelter](https://github.com/chainguard-dev/bom-shelter)
```

Comment:
~Chainguard has created aa repo called~
Another public repository for SBOMs in the wild is here